state/factory/workingset.go

```diff
@@ -279,12 +298,8 @@ func (ws *workingSet) Commit() error {
 
 // GetDB returns the underlying DB for account/contract storage
 func (ws *workingSet) GetDB() db.KVStore {
-	return ws.dao
-}
-
-// GetCachedBatch returns the cached batch for pending writes
-func (ws *workingSet) GetCachedBatch() db.CachedBatch {
-	return ws.cb
+	store, _ := db.NewKVStoreWithBuffer(ws.dao, ws.cb)
```

Comment:
return ws.kv?
or if intention is to return KVStoreWithBuffer, then return ws.dao?